codalab/rest/bundles.py

```diff
@@ -467,12 +467,12 @@ def _add_bundle_location(bundle_uuid: str):
     """
     check_bundles_have_all_permission(local.model, request.user, [bundle_uuid])
     need_bypass = query_get_bool('need_bypass', default=False)
-    is_dir = query_get_bool('is_dir', default=False)
+    is_dir = query_get_bool('is_dir', default=None)
```

Comment:
@wwwjn Can you add an assertion, before we save the bundle to the database, like `assert is_dir in (True, False)`? This way we can make sure it's never set to None.